go-controller/pkg/config/utils.go

```diff
@@ -50,11 +45,11 @@ func ParseClusterSubnetEntries(clusterSubnetCmd string) ([]CIDRNetworkEntry, err
 
 		entryMaskLength, _ := parsedClusterEntry.CIDR.Mask.Size()
 		if len(splitClusterEntry) == 3 {
-			tmp, err := strconv.ParseUint(splitClusterEntry[2], 10, 32)
+			tmp, err := strconv.Atoi(splitClusterEntry[2])
 			if err != nil {
 				return nil, err
 			}
-			parsedClusterEntry.HostSubnetLength = uint32(tmp)
+			parsedClusterEntry.HostSubnetLength = tmp
 
 			if ipv6 && parsedClusterEntry.HostSubnetLength != 64 {
```

Comment:
:)

Reply:
(OVN actually enforces this; it won't do IPAM for any IPv6 subnet size other than /64.)

Reply:
yeah, good, just I found it funny compared with the kubernetes one